src/olympia/addons/migrations/0026_addonregionalrestrictions.py

```diff
@@ -26,6 +26,6 @@ class Migration(migrations.Migration):
                 'abstract': False,
                 'base_manager_name': 'objects',
             },
-            bases=(olympia.amo.models.SearchMixin, olympia.amo.models.SaveUpdateMixin, models.Model),
+            bases=(olympia.amo.models.SaveUpdateMixin, models.Model),
```

Comment:
I wonder if it's a problem changing all these old migrations?  maybe we should just squash them?

Reply:
We've done this before, it should be fine (it's just to allow the schema editor to create realistic fake versions of those classes anyway)

Reply:
(I'd be up for a squash in the future though)